backend/spec/requests/journal/create_journal_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Mutations::CreateJournal, type: :request do
+  let(:mutation) { Mutations::CreateJournal.new(object: nil, field: nil, context: {}) }
+  let(:user) { FactoryBot.create(:user) }
```

Comment:
[imo]
rails_helperの記述のおかげでクラス名が省略できるようになってます🙋‍♂️

```suggestion
  let(:user) { create(:user) }
```

https://github.com/yuki-snow1823/diary/blob/ba5b21ee23fd4ec2b35b35b0d9f38f38880e0f17/backend/spec/rails_helper.rb#L64